src/css/main.less

```diff
@@ -473,20 +476,22 @@ fieldset[disabled] .btn-info.active {
 }
 
 .nav-tabs > li > a {
-  color: #8e929b;
+  color: @navbar-inactive-color;
   border-top: none;
   border-left: none;
   border-right: none;
   border-radius: 0;
+  min-width: @navbar-min-width;
   padding-bottom: 13px;
```

Comment:
Ok, fair enough.
My thought was to do it here, because this is directly related to the navbar.